src/transformers/modeling_gguf_pytorch_utils.py

```diff
@@ -242,7 +242,7 @@ def reverse_reshape_bias(weights: np.ndarray, n_head: int, n_embed: int):
 
 
 def split_moe_expert_tensor(
-    weights: np.ndarray, parsed_parameters: dict[str, dict], name: str, tensor_key_mapping: dict
+    weights: np.ndarray, parsed_parameters: Dict[str, Any], name: str, tensor_key_mapping: dict
```

Comment:
Not sure, but is it intended to be `Any`?

Reply:
Pushed a commit, this should be more accurate now